release/src/Main.hs

```diff
@@ -70,8 +72,19 @@ main = do
               -- set variables for next steps in Azure pipelines
               liftIO . putStrLn $ "##vso[task.setvariable variable=has_released;isOutput=true]true"
               liftIO . putStrLn . T.unpack $ "##vso[task.setvariable variable=release_tag]" # renderVersion sdkVersion
-          else $logInfo "Make dry run of release"
-
+          else if optsLocallyInstallJars
+                  then liftIO $ do
+                      forM_ (filter (\(mvn,_) -> artifactType mvn == "jar") uploadArtifacts) $ \(mvn_coords, path) -> do
+                          let args = ["install:install-file",
+                                      "-Dfile=" <> pathToString releaseDir <> pathToString path,
+                                      "-DgroupId=" <> foldr (<>) "" (List.intersperse "." $ map T.unpack $ groupId mvn_coords),
+                                      "-DartifactId=" <> (T.unpack $ artifactId mvn_coords),
+                                      "-Dversion=0.0.0-SNAPSHOT",
+                                      "-Dpackaging=" <> (T.unpack $ artifactType mvn_coords)]
+                          (_, Just hout, _, _) <- createProcess (proc "mvn" args) {std_out = CreatePipe}
```

Comment:
I don’t understand what you are trying to do here. Why capture stdout only to then print it again via `putStrLn`? I would just use `callProcess` which will also make sure to throw an exception if the process exits with a non-zero exit code.

Reply:
I guess I'm trying to do exactly what `callProcess` does. Will update to use that.